_layouts/home.html

```diff
@@ -1,57 +1,42 @@
 <!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link href="https://fonts.googleapis.com/css?family=Vollkorn" rel="stylesheet">
-    <link rel="stylesheet" href="/css/style.css">
-    <title>{{site.title}}</title>
+  <meta charset="UTF-8">
```

Comment:
The head tag for default and home layout are exactly same. Let's extract this into a partial file.